lib/measures/tbd/resources/psi.rb

```diff
@@ -1085,12 +1554,24 @@ def processTBD(os_model, psi_set, io_path = nil, schema_path = nil, gen_kiva)
 
   populateTBDdads(t_model, shades)
 
+  # Revise opaque surface net area (maybe subsurfaces with Frame & Divider).
+  surfaces.values.each do |p|
+    subarea = 0
+    if p.has_key?(:windows)
+      p[:windows].values.each   do |o| subarea += o[:gross]; end
+    end
+    if p.has_key?(:doors)
+      p[:doors].values.each     do |o| subarea += o[:gross]; end
+    end
+    if p.has_key?(:skylights)
+      p[:skylights].values.each do |o| subarea += o[:gross]; end
+    end
+    p[:net] = p[:gross] - subarea
+  end
+
```

Comment:
An opaque surface's net area is no longer API-reported - relying on Frame & Divider workarounds.